src/Apps/FeatureAKG/Components/Feature.tsx

```diff
@@ -173,24 +176,38 @@ const Section: React.FC<SectionProps> = props => {
   )
 }
 
-const VideoSection: React.FC<{ src: string }> = props => {
-  // Constrain the height, add 1px for the border
+const Video: React.FC<{ src: string }> = props => {
   return (
-    <Box mb="-40px" height="591px">
-      <SectionSeparator mt="4" />
-      <Box textAlign="center">
-        <Video src={props.src} />
-      </Box>
-    </Box>
+    <StyledVideo
```

Comment:
The header is a big mp4 video that they're going to compress as much as possible.

I'm worried about performance here-- do you have any suggestions?

Reply:
Generally we have three levers here:

1. Make it smaller
2. Transfer it faster
3. Load it later

I don't have a ton of recommendations. So long as it's being sent from a CDN, the only other thing I can really thing of is to defer its download until later. The loading experience might be hard to get right, but that's worth thinking about anyway.